internal/connector/pingone/mfa/resources/pingone_mfa_device_policy.go

```diff
@@ -14,57 +14,79 @@ var (
 )
 
 type PingOneMFADevicePolicyResource struct {
-	clientInfo *connector.PingOneClientInfo
+	clientInfo   *connector.PingOneClientInfo
+	importBlocks *[]connector.ImportBlock
```

Comment:
Why store the importBlocks on the resource vs just returning them directly from the ExportAll method? This feels a bit weird to me, since in theory if the same resource struct ran multiple imports the resulting import blocks would be duplicated.

Reply:
refactored!